src/input.F

```diff
@@ -306,6 +307,7 @@ subroutine inp_init
       calc_helium = ispval
       opdiffcap = spval
       electron_heating = ispval
+      enforce_n2 = .false.
```

Comment:
add a comment to explain the meaning of the knob?

Reply:
There's a description above where he defines the variable.
"! logical to correct the N2 altitude profile"